client/src/views/common/BoostView.tsx

```diff
@@ -117,7 +117,6 @@ export default function BoostView() {
         return config;
       }),
     );
-    toast.success(`${configName.displayName} deleted`);
```

Comment:
Removing this because it assumed everything worked well on the BOOST end and just blindly raised the `toast` without checking if the file was deleted or not.